assets/js/collectionPictures.js

```diff
@@ -1,41 +1,41 @@
 const getClosest = (elem, selector) => {
   for (; elem && elem !== window; elem = elem.parentNode) {
-    if (elem.matches(selector)) return elem
+    if (elem.matches(selector)) return elem;
   }
-  return null
+  return null;
 }
 
 const applyRemovePictureButtonEvent = pictureButtonElt => {
-  pictureButtonElt.addEventListener('click', e => {
-    let container = getClosest(e.target, 'fieldset')
+  pictureButtonElt.addEventListener("click", (e) => {
+    let container = getClosest(e.target, "fieldset");
 
     if (container !== null) {
-      container.remove()
+      container.remove();
     }
-  })
+  });
 }
 
-Array.from(document.getElementsByClassName('delete-picture-btn')).forEach(el => {
-  applyRemovePictureButtonEvent(el)
-})
+Array.from(document.getElementsByClassName("delete-picture-btn")).forEach(el => {
+  applyRemovePictureButtonEvent(el);
+});
```

Comment:
Codacy has a fix for the issue: [Replace `}` with `··}⏎`](https://app.codacy.com/gh/Florimond-Jouffroy/Openclassrooms-Project-006/pullRequest?prid=9522765)
```suggestion
  }
```